server/routes/nl.py

```diff
@@ -108,11 +108,13 @@ def _remove_places(query, places_str_found: List[str]):
   return ' '.join(query.split())
 
 
-def _get_place_from_dcids(place_dcids: List[str]) -> List[Place]:
+def _get_place_from_dcids(place_dcids: List[str]) -> Tuple[List[Place], Dict]:
```

Comment:
nit: personally I slightly preference to passing in auxiliary dbg stuff as an (in/out) argument or attached to a data structure (like Detection), esp for python. I feel it tends to be less error prone (eg if we miss changing a return site), and less intrusive to the main logic flow.

Reply:
modified in spirit of your suggestion